README.rst

```diff
@@ -2,121 +2,75 @@
-
-       $ pip install openslides-x.x.tar.gz
-
-   OpenSlides will install all required Python packages (see
-   requirements_production.txt).
+Installation
+============
 
+1. Installation on GNU/Linux or Mac OS X
+----------------------------------------
 
-Installation on Windows
------------------------
+a. Check requirements
+'''''''''''''''''''''
 
-*Note: There is a portable version of OpenSlides for Windows which does not
-require any install steps. If there is a reason that you can not use the
-portable version you should observe the following install steps.*
+Make sure that you have installed `Python <https://www.python.org/>`_ (>=
+3.4) on your system. You will also need the Python development headers (e.
+g. ``python3-dev`` for Ubuntu).
 
-1. Check requirements
+TODO: Check Ubuntu 14.04 and extra packages like libjpeg ...
 
-   Make sure that you have installed Python Programming Language 3 (>= 3.4)
-   and Setuptools on your system.
 
-   a. Download and run the latest `Python 3.5 32-bit (x86) executable
-      installer <https://www.python.org/downloads/windows/>`_. Note that
-      the 32-bit installer is required even on a 64-bit Windows system. If
-      you use the 64-bit installer, step 3 of this instruction will fail
-      unless you installed the package Reportlab manually.
+b. Setup a virtual Python environment (optional)
+''''''''''''''''''''''''''''''''''''''''''''''''
 
-   b. Download and run (via double click) the last `install script
-      ez_setup.py for Setuptools
-      <https://pypi.python.org/pypi/setuptools/#installation-instructions>`_.
+You can setup a virtual Python environment using the virtual environment
+(venv) package for Python to install OpenSlides as non-root user.
 
-2. Setup a virtual Python environment using the virtual environment (venv)
-   package for Python (optional)
+*Note: For Ubuntu 14.04 you have to install the pyvenv binary package*
+``python3.4-venv`` *before.*
 
-   You can setup a virtual Python environment to install OpenSlides as
-   non-root user.
+Create your OpenSlides directory, change to it, setup and activate the
+virtual environment::
 
-   Create your OpenSlides directory, change to it, setup and activate the
-   virtual environment::
+    $ mkdir OpenSlides
+    $ cd OpenSlides
+    $ python3 -m venv .virtualenv
+    $ source .virtualenv/bin/activate
 
-       > md OpenSlides
-       > cd OpenSlides
-       > python -m venv .virtualenv
-       > .virtualenv\Scripts\activate.bat
 
-3. Install OpenSlides
+c. Install OpenSlides
+'''''''''''''''''''''
 
-   You can use the package from the `OpenSlides Website
-   <http://openslides.org/download/>`_. Download latest OpenSlides release
-   as compressed tar archive and run::
+You can use the package from the `OpenSlides Website
+<http://openslides.org/download/>`_. Download latest OpenSlides release as
+compressed tar archive and run::
 
-       > pip install openslides-x.x.tar.gz
+    $ pip install openslides-x.x.tar.gz
```

Comment:
We should better use `pip install openslides` (from PyPi). Mention pip install *.tar.gz as alternative only.